platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -577,11 +619,30 @@ private void reportHasErrorMessage(Stream<MigrationException> errors, Matcher<St
         equalTo(1L));
   }
 
-  private ConfigurationMigrationManager getConfigurationMigrationManager() throws IOException {
+  private void createBrandingFile(String branding) throws IOException {
+    final File brandingFile = new File(ddfHome.resolve("Branding.txt").toString());
+
+    brandingFile.createNewFile();
+    Files.write(brandingFile.toPath(), branding.getBytes(), StandardOpenOption.APPEND);
```

Comment:
❓ Should we be passing an explicit charset into getBytes?

Reply:
For the purpose of this test, it will default to the default charset which is what we would be passing in anyway. We are also using the default charset to read it back.